package.json

```diff
@@ -38,8 +39,9 @@
     "redis": "^3.0.2",
     "setlistfm-js": "^1.0.1",
     "tslint": "^6.0.0",
-    "typescript": "^3.7.5",
-    "uuid": "^3.4.0"
+    "typescript": "^4.2.3",
```

Comment:
I had to bump Typescript to v4 for the AJV library. It didn't seem to cause any problems with anything else - apparently the v4 major version didn't come with breaking changes (somewhat surprisingly)

Reply:
👍🏼 thanks